syntax = "proto3";
package pb;

message TempEvent {
    int32 device_id = 1;
    int32 event_id = 2;
    float humidity = 3;
    float temp_cel = 4;
    bytes video_data = 5;
}